app/_landing_pages/konnect-authentication.yaml

```diff
@@ -0,0 +1,108 @@
+metadata:
+  title: "{{site.konnect_short_name}} authentication"
+  content_type: landing_page
+  description: |
+    Secure your {{site.konnect_saas}} organization by setting up teams and roles,
+    or enable an external authenticator to manage
+    {{site.konnect_saas}} authentication and authorization from your own identity
+    provider.
+  tags:
+    - konnect
+    - authentication
+  breadcrumb: /konnect-platform/
+    
+rows:
+  - header:
+      type: h1
+      text: "{{site.konnect_short_name}} authentication"
+      sub_text: |
+        {{site.konnect_saas}} offers multiple options for authentication. You can configure single sign-on with any SAML and OIDC provider or use built-in authentication.
+  - columns:
+      - blocks:
+        - type: structured_text
+          config:
+            header:
+              text: "Built-in authentication through {{site.konnect_short_name}}"
+            blocks:
+              - type: text
+                text: |
+                  The default authentication option in {{site.konnect_saas}} is basic authentication, which is enabled by default. 
+                  
+                  Users can also register for and sign in to Konnect accounts using their social identity from the following providers:
+                  * Google
+                  * GitHub
+                  * Microsoft
+
+                  The only way to disable social login is to [enable OIDC single sign-on](/konnect-platform/konnect-sso/) and disable the built-in authentication scheme. 
+                  This will also disable basic auth logins and delegate user management to the identity provider.
```

Comment:
```suggestion
```
These sentences feels more appropriate for a reference doc and I think we can remove it.